README.md

```diff
@@ -99,22 +99,22 @@ and headers (the header of the csv files ) as array of string.
 ```
 ParseDirectoryFiles(directoryPath, headers)
   => csvToJson(directory, file, headers)
-      => assign(fileName, maxEntriesPerFile, numberOfFiles, dataEntries)
-        => generate(i, fileName, start, stop, dataEntries)
+      => assign(fileName, dataEntries)
+        => generate(i, fileName, data)
 ```
 
 `ParseDirectoryFiles` gets a directory path from call, and reads all files in the
 directory but will only pass csv files to `csvToJson(directory, file, headers)`.
 Each csv file is passed into `csvParser()``.
 
 #### `csvToJson()` require csv-Parser modules`
-Fill `result` array with all csv entries => Total entries in csv file/10,000 entries per json file => gets number of json files to be generated => store in `numberOfFiles` => Pass file name, max entries per file, number of files, and csv data array to `assign`
```

Comment:
not sure why we don't using part of a content that we have at readme before.
I mean the more data we have, it'll be better for our team mates.
right nowi i'm talking about " Total entries in csv file/10,000 entries per json file => gets number of json files to be generated" - can you update it and add to your new line?

Reply:
@zhangted please update this line as well

Reply:
Okay